### ISSUE TITLE
too many SFTP connections

### ISSUE THREAD

**Issue by stefanunz** [mkloubert/vscode-deploy-reloaded#14]
### Description
when using deploy-reloaded with the below config, i get more and more stale sftpd processes on the target machine (I am frequently saving)
when I close VSC the stale sftd processes go away

I am deploying to a Raspberry Pi so memory consumption is a problem

### Actual behavior
deploy-reloaded opens more and more sftp connections to the target machine and does not close them

### Expected behavior
close sftp connection 
or
reuse sftp connection

#### Example config
```json
{
    "deploy.reloaded": {
        "packages": [
            {
                "name": "xgpioPack",
                "description": "Package version 2.3.4",

                "files": [
                    "/*.ts",
                    "/*.json",
                    "build/**"
                ],
                "exclude": [
                    "node_modules/**"
                ],

          
                "deployOnChange":[ "xgpioTar" ]
            }
        ],

        "targets": [
            {
                "type": "sftp",
                "name": "xgpioTar",
                "description": "A SFTP folder",

                "host": "10.2.x.x", 
                "port": 22,
                "user": "stefan", 
                "password": "x",

                "dir": "/x"
            }
        ]
    }
}
```

### Your environment

- Operating system: ubuntu x64 16.04
- Visual Studio Code version: 1.19.2
- Extension version: 0.29.0

**Comment by mkloubert** [comment]
@stefanunz 

**I have released a new version 0.30.3**, which should have a better handling of auto deploying files to a target at the same time.

**Comment by stefanunz** [comment]
ok thanks for the quick reaction, will retest soon ...
